sdk/ai/Azure.AI.Projects/tests/DatasetsTest.cs

```diff
@@ -0,0 +1,208 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using Azure.Identity;
+using NUnit.Framework.Internal;
+using Azure.AI.Projects.Tests.Utils;
+
+namespace Azure.AI.Projects.Tests
+{
+    // TODO: all tests as async by default
+    public class DatasetsTest : ProjectsClientTestBase
+    {
+        public DatasetsTest(bool isAsync) : base(isAsync)
+        {
+            TestDiagnostics = false;
+        }
+
+        [RecordedTest]
+        [Ignore("Calls to Azure.Storage.Blobs are not recorded with SCM structure")]
```

Comment:
This doesn't sound like something that would change, so why do we have this test, if it will never run?